layouts/partials/commons/logo.html

```diff
@@ -1,22 +1,50 @@
-{{ $logo := . }}
-{{ $file := false}}
+<!-- Logo(s) -->
+{{ $file := false }}
 {{ $file_darkmode := false }}
-{{ if eq (printf "%T" $logo) "string" }}
-  {{ $file = $logo }}
+{{ if eq (printf "%T" .logo) "string" }}
+  {{ $file = .logo }}
 {{ else }}
-  {{ $file = $logo.default }}
-  {{ $file_darkmode = $logo.darkmode }}
+  {{ $file = .logo.default }}
+  {{ $file_darkmode = .logo.darkmode }}
 {{ end }}
 
-<a class="logo {{- if $file_darkmode }} with-darkmode {{ end -}}" href="{{ site.Home.Permalink }}" title="{{ i18n "commons.index" (dict "Title" site.Title )}}">
+<!-- Title -->
+{{ $title := site.Title }}
+{{ if .title }}
+  {{ $title = .title }}
+{{ end }}
```

Comment:
J'utiliserai un `default` plutôt qu'une condition, c'est plus lisible

```suggestion
{{ $title := .title  | default site.Title }}
```